apps/ai-content-generator/src/configs/ai/gptModels.ts

```diff
@@ -1,10 +1,19 @@
-const gptModels = [
-  { id: 'gpt-3.5-turbo', name: 'GPT-3.5', textLimit: 12000 },
-  { id: 'gpt-3.5-turbo-16k', name: 'GPT-3.5 16k', textLimit: 12000 },
-  { id: 'gpt-4', name: 'GPT-4' },
-  { id: 'gpt-4-32k', name: 'GPT-4 32k' },
-];
+import OpenAI from 'openai';
 
-const defaultModelId = gptModels[0].id;
+async function getGptModels(apiKey: string) {
+  const openai = new OpenAI({ apiKey, dangerouslyAllowBrowser: true });
```

Comment:
I'm just curious, this looks different from how we are calling the api in other parts of the app, adding `dangerouslyAllowBrowser: true` and then calling openai.models.list vs using the modelsBaseUrl (https://github.com/contentful/apps/blob/master/apps/ai-content-generator/src/configs/ai/baseUrl.ts). Is this due to the version bump of the openai package?

Reply:
Will get rid of